rosserial_server/include/rosserial_server/session.h

```diff
@@ -116,6 +124,11 @@ class Session : boost::noncopyable
 
   void stop()
   {
+    std::cout << "stop!!!" << std::endl;
+    // Send stop tx command to MCU
+    //std::vector<uint8_t> message(0);
+    //write_message(message, rosserial_msgs::TopicInfo::ID_TX_STOP);
```

Comment:
Please clean this up, and replace the `cout` with {{ROS_INFO}} or {{ROS_DEBUG}} as appropriate.

Reply:
Revised in next commit.